recipe/custom_toolchain/cc_toolchain_config.bzl

```
@@ -304,15 +304,16 @@ def _impl(ctx):
     else:
         cxx_builtin_include_directories = [
             "${CONDA_BUILD_SYSROOT}/usr/include",
-	    "${BUILD_PREFIX}/lib/gcc/${HOST}/${COMPILER_VERSION}",
-	    "${BUILD_PREFIX}/${HOST}/include/c++/${COMPILER_VERSION}",
+            "${BUILD_PREFIX}/lib/gcc/${HOST}/${COMPILER_VERSION}",
+            "${BUILD_PREFIX}/${HOST}/include/c++/${COMPILER_VERSION}",
             "${PREFIX}/include",
+            "/usr/local/cuda/include",
```

Shouldn't this be,
```suggestion
            "${CUDA_HOME}/include",
```
and then `CUDA_HOME` substituted in `apply_cc_template`?

---

Yes, I like the suggestion. However, I've just completed building all the packages :)

If we could merge this first, then we can add this either for `2.7` or one of the other pending rebuilds?

---

I included a similar change